backend/db.py

```diff
@@ -200,20 +200,20 @@ def delete_file(self, file_url):
 
 # get rooms for a particular user
 def get_rooms(user_id, org_id):
-    """Get the rooms a user is in
-
-    Args:
-        user_id (str): The user id
-
-    Returns:
-        [List]: [description]
     """
+    This is a utility function that gets all the rooms linked to a user
+    It takes in two Args:
```

Comment:
Please, quick changes for the docstring. You can follow the example below
`"""Fetches rows from a Smalltable.

    Retrieves rows pertaining to the given keys from the Table instance
    represented by table_handle.  String keys will be UTF-8 encoded.

    Args:
        table_handle: An open smalltable.Table instance.
        keys: A sequence of strings representing the key of each table
          row to fetch.  String keys will be UTF-8 encoded.
        require_all_keys: If True only rows with values set for all keys will be
          returned.

    Returns:
        A dict mapping keys to the corresponding table row data
        fetched. Each row is represented as a tuple of strings. For
        example:

        {b'Serak': ('Rigel VII', 'Preparer'),
         b'Zim': ('Irk', 'Invader'),
         b'Lrrr': ('Omicron Persei 8', 'Emperor')}

        Returned keys are always bytes.  If a key from the keys argument is
        missing from the dictionary, then that row was not found in the
        table (and require_all_keys must have been False).

    Raises:
        IOError: An error occurred accessing the smalltable.
    """`